script.lua

```diff
@@ -5866,7 +5844,10 @@ function setupPlayerArea(params)
                                 elements:add(cardElements)
                                 nonTokenElements:add(cardElements)
                             end
-                            energy = energy + powerCost(entry)
+                            -- Skip counting locked card's energy (Aid from Lesser Spirits)
+                            if not entry.getLock() then
+                                costs[entry.guid] = entry.getVar("energy")
```

Comment:
might be worth still having the `entry.getVar("energy") == nil` check somewhere around here even though it should never happen

Reply:
The code's actually safe against that happening. Lua treats a key set to a value of `nil` as equivalent to not being in the table. So when we iterate over that table with `pairs()` (as when we sum the costs), it will ignore the entry.

I can put the check in explicitly if you think it's better, but it'd be purely for readability; it wouldn't affect functionality even if `entry.getVar("energy")` did return `nil`.

Reply:
ah yeah true, should be fine as is